go.mod

```diff
@@ -124,7 +124,7 @@ require (
 	github.com/hashicorp/vault-plugin-database-mongodbatlas v0.9.0
 	github.com/hashicorp/vault-plugin-database-redis v0.2.0
 	github.com/hashicorp/vault-plugin-database-redis-elasticache v0.2.0
-	github.com/hashicorp/vault-plugin-database-snowflake v0.7.3
+	github.com/hashicorp/vault-plugin-database-snowflake v0.7.0
```

Comment:
Odd that we are downgrading this and some azure indirect dependencies below?

Reply:
oh yeah, weird.  Let me take another crack at this one.